tests/system/action/meeting/test_import.py

```diff
@@ -1742,7 +1765,7 @@ def test_all_migrations(self) -> None:
         with CountDatastoreCalls(verbose=True) as counter:
             response = self.request("meeting.import", data)
         self.assert_status_code(response, 200)
-        assert counter.calls == 6
+        assert counter.calls == 7
```

Comment:
This is the test to test all migrations. Please remove the organization_id in instance data (`del data["meeting"]["user"]["1"]["organization_id"]` and insert a test that it was set by migration (`self.assert_model_exists("organization/1", {"user_ids": [1, 2], "active_meeting_ids": [1, 2]})`)

Reply:
Done.